### ISSUE TITLE
add account based access to file list

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry#2159]
## The Issue
Similar to other commands that take account_id as a parameter, this should also be possible with file list. This will allow services like lbry.tv to use the SDK for streaming and keep file / payment information for customers.  

This ticket would encompass any database / api changes required to do so.  Should take into account the scenario where blobs/files are shared between accounts so that data is not duplicated on disk. 

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by sayplastic** [comment]
I think `get` command should accept `account_id` as well.

**Comment by lyoshenka** [comment]
we all discussed this on app team standup on wednesday. next step is jeremy/sean will get us more details on how the app uses file_list api

**Comment by tzarebczan** [comment]
From Slack/Jeremy / thread: https://lbryians.slack.com/archives/C81K0BM9A/p1560807046184800

```
Here are my notes on Library / History functionality as it relates to `file_list` and `stream_list`, and agreed upon by app team.

https://docs.google.com/document/d/1_xaUYOOg-KptEHoSTCvofu9xYoUbEX4ywtfxC7BajTc/edit#

tl;dr:
1) I do not think `file_list` will need an account flag, but do check my assumptions/logic
2) We will need an API call to just list purchased claims
3) Key/value storage in wallet would be useful because sync service can then sync "my library"
```

**Comment by kauffj** [comment]
Conclusions of discussion with @lbryio/appsquad about how the app will use purchase data:
- The app will be displaying information about purchases regularly when rendering claim data. This would include results from `claim_search` as well as when we're calling `resolve` (which is basically `claim_show`). If calls that return claims cannot include a sub-field like `purchase_txid` or `purchase_outpoint`, then the app will end up needing to load _all_ purchases at load.
- Apps will not be co-mingling purchases with files on disk. Instead, the Library section will be split into separate areas (downloaded vs. purchased). However, it will be rendering purchases the same way it renders claims. So if we're not being given the full claim when listing purchases, we'll end up needing to make an extra `resolve` to return the claim IDs we are given.

**Comment by eukreign** [comment]
- `claim_search --name=foo --wallet_id=xyz123` and `resolve lbry://foo --wallet_id=xyz123` will check the respective wallets if there is a purchase receipt for the returned claims and add a `purchase_receipt` value to the response, if found.
- `purchase_list --resolve --wallet_id=xyz123` will return purchases with purchased claims resolved and placed in the `claim` field (without `--resolve` it will not perform the additional resolves for each purchase and only return the purchase data).
- `transaction_list` adds a new `purchase_info` section with purchase info.
